jenkins/common.groovy

```diff
@@ -115,6 +115,10 @@ def archive(dockerContainer, String ref, String envName) {
     withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'vetsgov-website-builds-s3-upload',
                      usernameVariable: 'AWS_ACCESS_KEY', passwordVariable: 'AWS_SECRET_KEY']]) {
       sh "echo \"${buildDetails}\" > /application/build/${envName}/BUILD.txt"
+
+      sh "cd /application && yarn test:coverage-only"
+      sh "node script/app-coverage-report.js"
```

Comment:
The app-coverage-report script doesn't copy to the build directory anymore, if that was the expected behavior here. It's in /coverage, so it'll need to be copied from there or modify the app-coverage-report script again. One more example of us coming full circle with this ticket haha.

Reply:
Good catch, thank you!

Reply:
Done!